### DISCUSSION TITLE
Azure API Management OData Service not showing up in Developer Portal

### DISCUSSION THREAD

**Discussion by HPatArla** [Azure/api-management-developer-portal#2630] [Feature request]
Im having the same problem as discribed in https://stackoverflow.com/questions/78570696/azure-api-management-odata-service-not-showing-up-in-developer-portal.:  

"I've imported an OData Service as an API in Azure API Management by following the steps in this official article: https://learn.microsoft.com/en-us/azure/api-management/import-api-from-odata

I expected the OData to show up in de APIM Developer Portal, but it isn't listed in de Api's page. I have all the permissions i need, and the OData Service API is also added to a product. The strange things is that when i navigate to the Products page, and select the product, the OData API is listed as an API of that product. But when i navigate to the API from here, it goes to the API page, but it says "The specified API does not exist"."

here is my example:

![image](https://github.com/Azure/api-management-developer-portal/assets/85360270/8fd65f89-1cc7-43fe-af07-7f6311df9ab7)
![image](https://github.com/Azure/api-management-developer-portal/assets/85360270/a95ef377-aaf6-4caa-986f-3098eb9f838f)

When will the Odata API's beeing supported in the developer Portal?

**Comment by sthirthala** [comment]
This is something we are working on

**Reply by oldDusty** [reply]
Hi sthirthala,

When can we expect OData APIs to be accessible from the Developer Portal?

We have added a few SAP OData APIs to APIM as part of an exploration phase but then we hit this limitation. Hopefully there's a date planned and it's not too far off!

**Comment by HPatArla** [comment]
Thanks, looking forward, and let me know if you need me to test early on :-)

**Comment by sivapokuri** [comment]
Do you happen to have any updates on this feature? I have the same issue. Please comment

**Comment by BahaAlAttia** [comment]
We installed APIM in production and I am surprised with the same issue. I am really very much interested in knowing when this issue will be resolved?

**Reply by ceesharkes** [reply]
We're experiencing the same issue and hope it gets resolved soon—our clients are eagerly awaiting a solution.
